composer.json

```diff
@@ -14,6 +14,7 @@
         }
     ],
     "require": {
+        "php": "^5.5 || ^7.0",
```

Comment:
`|` is the old. `||` is the new and npm compatible syntax ^^
See https://twitter.com/PatrikKarisch/status/705173701088694274